server/routes/referral/cancellation/confirmation/referralCancellationConfirmationPresenter.test.ts

```diff
@@ -11,8 +11,9 @@ describe(ReferralCancellationConfirmationPresenter, () => {
 
       expect(presenter.text).toMatchObject({
         confirmationText: 'This referral has been cancelled',
+        headingText: 'What you need to do next',
         whatHappensNextText:
-          "Service provider will be notified about the cancellation. You don't have to do anything else.",
+          'You need to contact the service provider outside the service to let them know about the change.',
```

Comment:
Can we include the address, to make this more actionable?

Reply:
I've updated this to add the email. Speaking with Helen because she wants to check this is the correct approach first before it is merged.

Reply:
The change made me realise that we don't actually have an intended "contact" email for providers, sorry 🤦
The `incomingReferralDistributionEmail` is invalid for 11 contracts...

Possibly something to pick up as a follow-up

Reply:
No problem! Can revert for this change, it was a useful learning experience adding that in if nothing else :)